### PR TITLE
Fix wrong binding for scalardb config in the compose files

### PR DESCRIPTION
**Pull request by thongdk8**
We should bind the config file to the `database.properties.tmpl` instead of `database.properties`. This PR is made to fix it. PTAL! Thank you!

### CODE REVIEW

File: scalardb-server-sample/docker-compose-cassandra.yml [hunk 1]
```diff
@@ -14,12 +14,15 @@ services:
...
-      - ./database.properties:/scalardb/server/database.properties
+      - ./database.properties:/scalardb/server/database.properties.tmpl
       - ./wait-for-it.sh:/scalardb/server/wait-for-it.sh
     ports:
       - "60051:60051" # scalardb-server port
     entrypoint: /bin/bash
-    command: ./wait-for-it.sh -t 60 cassandra:9042 -- ./bin/scalardb-server --config database.properties
+    command: >
+      ./wait-for-it.sh -t 60 cassandra:9042 --
+      dockerize -template database.properties.tmpl:database.properties -template log4j2.properties.tmpl:log4j2.properties --
```
Comment (kota2and3kan) [inline scalardb-server-sample/docker-compose-cassandra.yml:24]:
```suggestion
      dockerize -template database.properties.tmpl:database.properties -template log4j2.properties.tmpl:log4j2.properties
```
I think this double dash is not necessary.

The `dockerize` command works properly without a double dash as follows.
https://github.com/scalar-labs/scalardb/blob/master/server/Dockerfile#L57-L59

**Review by kota2and3kan — COMMENTED** [review]
Thank you for the update!
Overall looks good to me, but I left a minor suggestion.
Please take a look!

File: scalardb-server-sample/docker-compose-cassandra.yml (hunk 1 above)
Comment (thongdk8) [reply scalardb-server-sample/docker-compose-cassandra.yml]: Thank you for the suggestion. I applied it

**Review by kota2and3kan — APPROVED** [review]
LGTM! Thank you!

**Review by brfrn169 — APPROVED** [review]
LGTM! Thank you!

**Review by Torch3333 — APPROVED** [review]
LGTM, thank you!

**Pull request merged**